### PR TITLE
Implement reset method for ProfilerExtension

### PR DESCRIPTION
**Pull request by ahilles107**
Fix Symfony 3.4 depreciation

### CODE REVIEW

**Comment by sweoggy** [comment]
Related issue #20

**Comment by ahilles107** [comment]
@EmanueleMinotto - can you look at it?

File: Twig/ProfilerExtension.php [hunk 1]
```diff
@@ -140,4 +140,11 @@ public function unserialize($serialized)
             $this->strategyClass = $data['strategyClass'];
         }
     }
+
```
Comment (sweoggy) [inline Twig/ProfilerExtension.php]: Unnecessary white spaces

File: Twig/ProfilerExtension.php [hunk 2]
```diff
@@ -140,4 +140,11 @@ public function unserialize($serialized)
             $this->strategyClass = $data['strategyClass'];
         }
     }
+    
+    public function reset()
```
Comment (sweoggy) [inline Twig/ProfilerExtension.php]: Add PHPDoc

**Review by sweoggy — CHANGES_REQUESTED** [review]

**Comment by ahilles107** [comment]
@sweoggy fixed.

**Comment by Gemineye** [comment]
Any news?!

**Comment by mikemix** [comment]
Any news? :(

**Comment by ahilles107** [comment]
Hey, 

we wan't to use this bundle in Symfony >=4. Please look at it and merge if possible.

**Comment by sweoggy** [comment]
Might be worth creating an active fork, has been almost three months now.

**Pull request merged**